whitelabelling_bar/package.json

```diff
@@ -0,0 +1,130 @@
+{
+    "name": "whitelabelling_bar",
+    "version": "1.0.5",
```

Comment:
@dremileta we wanted to have the version in sync with the internal repo, however, we have here a newer version of the SDK, can you please update it there as well and bump thee patch version everywhere.

Reply:
@PavelJiranek you mean bum up internal to 1.0.6 and here too?

Reply:
Yep, the version should be the same as here: https://github.com/gooddata/dashboard-plugins-internal/pull/98/files#diff-c4bdcc0095a85fb3d723841890990b082d09be1445b7b1e14ca4ccbad44ca823R3  if the plugin code and deps are the same